produce-data.json

```diff
@@ -0,0 +1,43 @@
+{
+    "produce": [
+        {
+            "name": "Tomato",
+            "zones": [1, 2, 3, 4, 5, 6, 7, 8, 9, 10],
+            "startSeedIndoors": ["Feb 15", "March 15", "March 15", "March 15",
+             "March 15", "March 15", "March 15", "March 15", "March 15", "March 15"],
+            "transplantPlantOutdoors": ["May 20", "May 20", "May 20", "May 20", 
+            "May 20", "May 20", "May 20", "May 20", "May 20", "May 20"],
+            "plantCompanions": "Dill, Basil, Peppers..."
```

Comment:
this should also be a string array, will make getting this data visualized in your app a lot easier than just a single string.